fluentd/run.sh

```diff
@@ -31,6 +31,10 @@ if [ "${USE_JOURNAL:-}" != false ] ; then
     fi
 fi
 
+IPADDR4=`/usr/sbin/ip -4 addr show dev eth0 | grep inet | sed -e "s/[ \t]*inet \([0-9.]*\).*/\1/"`
```

Comment:
I see ipaddr4 but not ipaddr6 in pipeline_metadata.collector in com.redhat.viaq-openshift-project.template.json and com.redhat.viaq-openshift-operations.template.json.  I'm wondering if it is to be added to them or no need?